cmd/data-usage-cache.go

```diff
@@ -317,6 +374,16 @@ func (d *dataUsageCache) replace(path, parent string, e dataUsageEntry) {
 	}
 }
 
+// listCache will return all cache paths.
+func (d *dataUsageCache) listCache() []string {
+	dst := make([]string, 0, len(d.Cache))
+	for k := range d.Cache {
+		dst = append(dst, k)
+	}
+	sort.Strings(dst)
+	return dst
+}
```

Comment:
this function is not used. can be removed?

Reply:
Perhaps @klauspost do this in separate PR.